autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/ScalingRequest.java

```diff
@@ -156,14 +160,14 @@ private void scaleDown(int scalingAdjustment, int totalNodes) {
             cloudbreakCrnClient.withInternalCrn().autoscaleEndpoint()
                     .putCluster(stackCrn, cluster.getClusterPertain().getUserId(), updateClusterJson);
             scalingStatus = ScalingStatus.SUCCESS;
-            statusReason =  getMessageForCBStatus(ScalingStatus.SUCCESS.name());
+            statusReason = getMessageForCBSuccess();
             metricService.incrementMetricCounter(MetricType.CLUSTER_DOWNSCALE_SUCCESSFUL);
         } catch (Exception e) {
             scalingStatus = ScalingStatus.FAILED;
             metricService.incrementMetricCounter(MetricType.CLUSTER_DOWNSCALE_FAILED);
-            statusReason = getMessageForCBStatus(e.getMessage());
+            statusReason = getMessageForCBException(e);
             LOGGER.error("Couldn't trigger downscaling for host group '{}', cluster '{}', desiredNodeCount '{}', error '{}' ",
-                    hostGroup, cluster.getStackCrn(), desiredNodeCount, e);
+                    hostGroup, cluster.getStackCrn(), desiredNodeCount, statusReason, e);
```

Comment:
slf4j handles Exceptions without requiring a separate '{}' in the log message, correct?

Reply:
slf4j logs exception stack trace without '{}' , but doesn't extract server error text from BadRequest and similar exceptions.